_data/errors.yml

```diff
@@ -83,11 +83,39 @@ oidc:
         - Ensure the `redirect_uri` that you are sending as part of your authentication request is registered. You must register every redirect uri that your application redirects through.
     - title: Unauthorized scope
       id: oidc-unauthotized-scope
-      accordion-id: oidc-unauth-scope
+      accordion-id: oidc_unauth_scope
       content: |
         ##### Why it's happening
         You have requested attributes that your Service Level does not allow.
         ##### What to do:
         - Check the `scope` parameter in your authentication request.
         - If your application is an *Authentication Only* application, ensure you are only passing in IAL1 attributes.
         More information about the different types of attributes can be found here: [https://developers.login.gov/attributes/](https://developers.login.gov/attributes/){:target="_blank"}
+  logout:
+    - title: client_id was not recognized
+      id: client-id-not-recognized
+      accordion-id: client_id_not
+      content: |
+        ##### Why it's happening
+        The issuer (also known as the client id) provided does not match a service provider registered in the IdP.
+        ##### What to do:
+        - Ensure the logout request has a `client_id` value.
+        - Ensure that the `client_id` sent in the logout request matches the one registered in your application on the [Partner Dashboard](https://dashboard.int.identitysandbox.gov/){:target="_blank"}.
+        - Ensure that the application configuration's `active` value is set to `true`.
+    - title: client_id is missing
+      id: client-id-missing
+      accordion-id: client_id_missing
+      content: |
+        ##### Why it's happening
+        The issuer (also known as the client id) was not found.
+        ##### What to do:
+        - Ensure the logout request has a `client_id` value.
+    - title: This application is misconfigured and should not be sending id_token_hint. Please send client_id instead.
+      id: id-token-hint
+      accordion-id: id_token_hint
+      content: |
+        ##### Why it's happening
+        We are no longer supporting the id_token_hint attribute as a method of logging out.
+        ##### What to do:
+        - Update your logout request to utilize the client_id attribute.
+        More information about the logout request parameters can be found here: [https://developers.login.gov/oidc/#logout-request](https://developers.login.gov/oidc/#logout-request){:target="_blank"}
```

Comment:
Please add a period at the end of this paragraph

Reply:
d'oh i thought i caught them all

Reply:
fixed, thanks!